TenantFile/Api/DataLoader/AddressByIdDataLoader.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using GreenDonut;
+using HotChocolate.DataLoader;
+using Microsoft.EntityFrameworkCore;
+using TenantFile.Api.Models;
+using TenantFile.Api.Models.Entities;
+
+namespace TenantFile.Api.DataLoader
+{
+    public class AddressByIdDataLoader : BatchDataLoader<int, Address>
```

Comment:
Why use dataloaders? Unless I'm missing something, we don't need to use dataloaders because we use an ORM.